atomic_reactor/plugins/pre_distribution_scope.py

```diff
@@ -89,11 +88,11 @@ def run(self):
             scope = self.get_scope('current', labels)
 
             # Find out the parent's intended scope
-            # OSBS2 TBD
-            inspect = imageutil.base_image_inspect()
+            # OSBS2 TBD: decide if we need to inspect a specific arch
```

Comment:
distribution scope is set by osbs, so it will be same for all arches of the parent so we don't nee arch specific